spark/src/test/scala/org/apache/spark/sql/benchmark/CometAggregateBenchmark.scala

```diff
@@ -28,7 +28,7 @@ import org.apache.comet.CometConf
 /**
  * Benchmark to measure Comet execution performance. To run this benchmark:
  * {{{
- *   SPARK_GENERATE_BENCHMARK_FILES=1 make benchmark-org.apache.spark.sql.benchmark.CometAggregateBenchmark
+ *   make benchmark-org.apache.spark.sql.benchmark.CometAggregateBenchmark
```

Comment:
This looks like it was just an old config that wasn't doing anything anymore?

Reply:
Yes, that seems to be the case.

Reply:
This env variable is used in Spark's [BenchmarkBase](https://github.com/apache/spark/blob/97e0a88b1d04a8e33206e5cb8384878ca4cdfc5a/core/src/test/scala/org/apache/spark/benchmark/BenchmarkBase.scala#L51) which is the parent of this (and other benchmarks).
The effect is to write the output to a file under the `spark/benchmarks`.
In Spark, for any PR that has a performance impact, the contributor can run the benchmarks in their github environment and [include the benchmark results](https://spark.apache.org/developer-tools.html#github-workflow-benchmarks) as part of the PR. This enables reviewers to verify the performance against previously run benchmarks. Since the benchmarks are run on github, all users end up running the benchmarks on the same hardware and environment and so the results are actually comparable.
Note that this is also the reason why, once a benchmark case has been given a name, it should not be changed. The benchmark case has a name solely to allow this comparison.

Reply:
At some point, like Spark, we will enable this for Comet so that updates to benchmark results are added as part of the PR and then we can track changes to the benchmark results.

Reply:
Thanks @parthchandra. I had only searched within our repo trying to find where it was used. I have reverted removing these references.